README.md

```diff
@@ -218,3 +218,9 @@ gRPC comes with three Transport implementations:
    and is for client only.
 3. The in-process transport is for when a server is in the same process as the
    client. It is useful for testing, while also being safe for production use.
+
+Third-Party Integration
+-----------------------
+
+* [grpc-spring-boot-starter](https://github.com/yidongnan/grpc-spring-boot-starter)
```

Comment:
Just out of curiosity, I saw at least two grpc-spring-boot-starters on github: this one and https://github.com/LogNet/grpc-spring-boot-starter, what is the difference and their relation? Which one is more widely used?

Reply:
**Differences**

The two projects have two different philosophies.

* [LogNet/grpc-spring-boot-starter](https://github.com/LogNet/grpc-spring-boot-starter) is server only and focuses on being light weight
  (Size: ~15KB)
* [yidongnan/grpc-spring-boot-starter](https://github.com/yidongnan/grpc-spring-boot-starter) is for both server and client side and aims at full integration with spring, thus it also supports additional features such as Spring-Security and Actuator
  (Size: ~160KB; roughly ~90KB per side)

**Relation**

The only relation they have is their topic "combining gRPC with spring", thus I would call them competitors for lack of better names. I'm not aware of any other relationships between them.

**Usage**

I don't have any numbers. 

The following is very opinionated:
* LogNet is currently preferred by the Google Search. LogNet started earlier and does not have the eastern name.
* yidongnan has an increasing number of visitors and contributors lately.

Reply:
Thanks a lot for your explanation. As discussed in https://github.com/grpc/grpc.github.io/pull/832, I agree that [Awesome gRPC](https://github.com/grpc-ecosystem/awesome-grpc) is a better place for the link (already there) . You may also want to move/mirror your github repo to https://github.com/grpc-ecosystem, we can consider that too.